sphinx/source/conf.py

```diff
@@ -64,7 +64,7 @@
 
 autodoc_member_order = 'groupwise'
 
-bokeh_missing_google_api_key_ok = False
+bokeh_missing_google_api_key_ok = False #True   
```

Comment:
I am not sure this comment adds anything? Anyone who might need to change it will surely know that the value to use is True?

Reply:
@bryevdv before doing the final commit I just reverted back the changes, which I usually do by just commenting out the non-required boolean value. It can certainly be removed, I just kept it in thinking it doesn't do much harm!

Reply:
Well, I wouldn't hold up merging if you really feel strongly about it, but my own preference is definitely to remove it.

Reply:
Alright, will remove it!

Reply:
Removed it!